dig_test.go

```diff
@@ -2721,3 +2721,57 @@ func TestNewDotCtor(t *testing.T) {
 	assert.Equal(t, []*dot.Node{{Type: "dig.t1"}}, ctor.Params)
 	assert.Equal(t, []*dot.Node{{Type: "dig.t2"}}, ctor.Results)
 }
+
+func TestVisualize(t *testing.T) {
+	n1, n2, n3, n4 := &dot.Node{Type: "t1"}, &dot.Node{Type: "t2"}, &dot.Node{Type: "t3"}, &dot.Node{Type: "t4"}
```

Comment:
nit: putting these four on their own lines would be more readable. you don't gain much by putting it all on one line.